moabb/paradigms/p300.py

```diff
@@ -99,6 +99,14 @@ def process_raw(self, raw, dataset):
 
         # pick events, based on event_id
         try:
+            if type(event_id['Target']) == list:
+                if type(event_id['NonTarget']) == list:
```

Comment:
You could merge these two tests in one line using `if type .... and type ...`
Also, when testing identity of a type, it is best to use `is`, see these [recommandations](https://legacy.python.org/dev/peps/pep-0008/#programming-recommendations). You could thus write something like: `if type(event_id['Target']) is list and type...`.

Reply:
I implemented the change and removed the ValueError, which would be thrown if only one of the event_ids were passed as a list and merged the latest master into this branch.